app/models/UserProfile.ts

```diff
@@ -15,6 +15,16 @@ export default class UserProfile extends BaseModel {
     @Column({ nullable: true })
     public dob: Date;
 
+    /**
+     * Gender is specified as a single 4-bit character, with the range of 26 possible options with
+     * a total of 52 if being case insensitive. This will cover them for the base M and F with
+     * the option to support a larger range of inclusivity if possible..
+     *
+     * Null support for if the user does not want to provide there gender during creation.
+     */
+    @Column({ nullable: true, length: 1 })
```

Comment:
Use enum instead of a single 4-bit character to store the following options:
1. Male
2. Female
3. Other

and of course, keep nullable to allow the user to not provide their sex if they wish.

Reply:
This enum expected to be a in memory or referenced via the database? (will this exist in the database or not).

Reply:
Please reference GameSchedule.ts for an example of how to implement.